platform/linux-generic/odp_classification.c

```diff
@@ -1718,13 +1718,15 @@ int _odp_cls_classify_packet(pktio_entry_t *entry, const uint8_t *base,
 	if (cos->action == ODP_COS_ACTION_DROP)
 		return 1;
 
-	if (cos->queue == ODP_QUEUE_INVALID && cos->num_queue == 1)
-		goto error;
-
-	if (cos->pool == ODP_POOL_INVALID)
-		goto error;
+	if (cos->queue == ODP_QUEUE_INVALID && cos->num_queue == 1) {
```

Comment:
Git log entry text has copy-paste error: "CoS pool is ODP_QUEUE_INVALID" - pool vs. queue invalid